po/de.po

```diff
@@ -2194,15 +2197,18 @@ msgstr "@untouched Schlagwort hinzufügen"
 #: GTG/plugins/dev_console.gtg-plugin.desktop:4
 #: GTG/plugins/dev_console/console.py:71 GTG/plugins/dev_console/console.py:100
 msgid "Developer Console"
-msgstr ""
+msgstr "Entwickerkonsole"
```

Comment:
Typo here, should be "Entwicklerkonsole"

Reply:
How did I missed that? Fixed.